magento/magento2
# Fix wrong typo function name

### Description (*)

Correct function name caused by typo

### Related Pull Requests


### Fixed Issues (if relevant)

1. Fixes magento/magento2#<issue_number>

### Manual testing scenarios (*)

1. From admin dashboard
2. Go to catalog > Category
3. Go to workarounds new category field
4. All function should work without break

### Questions or comments


### Contribution checklist (*)


### Resolved issues:
1. [x] resolves magento/magento2#29987: Fix wrong typo function name
